server/src/test/kotlin/org/itmodreamteam/myrest/server/service/restaurant/RegisterRestaurantTest.kt

```diff
@@ -0,0 +1,54 @@
+package org.itmodreamteam.myrest.server.service.restaurant
+
+import org.assertj.core.api.Assertions.assertThat
+import org.itmodreamteam.myrest.server.error.UserException
+import org.itmodreamteam.myrest.server.model.restaurant.Restaurant
+import org.itmodreamteam.myrest.server.repository.restaurant.RestaurantRepository
+import org.itmodreamteam.myrest.server.service.user.SignInTest
+import org.itmodreamteam.myrest.shared.restaurant.RestaurantInfo
+import org.itmodreamteam.myrest.shared.restaurant.RestaurantStatus
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit4.SpringRunner
+
+@RunWith(SpringRunner::class)
+@DataJpaTest
+@ContextConfiguration(classes = [SignInTest.Config::class])
```

Comment:
It's better to use an independent config per test class. Create a nested class Config and change the reference.